assets/github/pipeline.tmpl

```diff
@@ -9,16 +9,22 @@ on:
 jobs:
   cli:
     runs-on: ubuntu-latest
-    container: 
-      image: ghcr.io/nearform/k8s-kurated-addons-cli:latest
     steps:
+      - uses: actions/checkout@v2
+
       - name: build docker image application
-        run: build
-      
+        uses: docker://ghcr.io/nearform/k8s-kurated-addons-cli:latest
```

Comment:
is there some benefit of using docker:// vs container.image property?

Reply:
Container Image doesn't work 😅

Reply:
I just found another solution that might work, I'll try it later